build/automation/var/platform-texas/v1/account-live-k8s-prod.mk

```diff
@@ -1,5 +1,8 @@
 AWS_ACCOUNT_ID := $(AWS_ACCOUNT_ID_PROD)
 AWS_ACCOUNT_NAME := prod
+AWS_CERTIFICATE = 8d65eee4-cf92-4a00-84de-7a9f544ba724
```

Comment:
Is this DoS CS API domain-specific certificate? It may need to be moved to project.mk - otherwise (if this is *.k8s-nonprod.texasplatform.uk) then that's all fine leaving it in here.

Reply:
Providing the above is correct, can we have a PR to be created for that change to the Make DevOps repository as this is the library file? Thanks

Reply:
Done